ceph/rados/rados_scrub.py

```diff
@@ -88,7 +88,7 @@ def get_pg_dump(self, *args):
         else:
             return pgDump
 
-    def verify_scrub(self, before_scrub_data, after_scrub_data):
+    def verify_scrub(self, before_scrub_data, after_scrub_data, flag):
```

Comment:
I feel it's good to separate out functionality verification of scrub & deep-scrub into different methods. 

If not, please update the method name, docstrings and method params to better reflect the combined testing of both scrub & deep-scrub.

Reply:
Updated the method name, doc strings, and params.